### ISSUE TITLE
QT interop with Kha

### ISSUE THREAD

**Issue by hsdk123** [KTXSoftware/Kha#467]
Hi, I've been loving Kha as of far and am currently looking to see if I could pitch it for use with my team / company. The only catch I see as of far though would be interop with QT to make editors / tools usable during development - is there anything along these lines that might be known of? Seems like I could really dive in with Kha if this was possible.

**Comment by RobDangerous** [comment]
Kha would need a new backend for this but it wouldn't be particularly hard to implement that (have a look at the Linux backend - would look kinda similar to that). Additionally you'd have to somehow integrate with Qt's build system.

**Comment by dmitryhryppa** [comment]
I'm currently searching ways to do the same thing. 
Currently I got haxe to working with QT as a static .lib

So, perhaps we can integrate Kha as well. 
I think we just need to remove window creation stuff and then pass custom control HWND to render Kha stuff on it.

![image](https://cloud.githubusercontent.com/assets/4466595/21721386/d4e035d2-d430-11e6-8b15-8b86f8c6e6bd.png)

**Comment by hsdk123** [comment]
@dmitryhryppa @RobDangerous Yup, that's nicely put. It would be great if Kha had a simple interface or option for rendering into external windows / window handles.

**Comment by RobDangerous** [comment]
That's something I can do, as I don't have to install Qt for it (it's so big and weird). I'll just split up the window creation code a little.
